src/main/java/duke/Ui.java

```diff
@@ -174,4 +163,34 @@ public void echoTaskToday(ArrayList<Task> taskList) {
         }
         System.out.println("    ____________________________________________________________\n");
     }
+
+    /**
+     * Print all the tasks that match the pattern of a find command.
+     *
```

Comment:
There seems to be an unnecessary spacing in your javadoc comment. Would it be better to take it out? I have noticed this in other javadoc comments as well.